modyn/protos/evaluator.proto

```diff
@@ -54,25 +58,25 @@ message EvaluationStatusRequest { int32 evaluation_id = 1; }
 message EvaluationStatusResponse {
   bool valid = 1;
   bool is_running = 2;
-  bool state_available = 3;
-  bool blocked = 4;
-  optional string exception = 5;
```

Comment:
Why do we need to remove these logging fields completely? At least `batches_seen` and `samples_seen` are quite nice to verify that evaluations actually contained the expected amount of data.

Reply:
Xianzhe asked whether we can remove the tracking but I thought we were talking about the visual progress bar. The amount of data in each interval we definitely need to track, this is useful debugging information

Reply:
yeah these few fields are useful visually, to report the progress of the evaluation while it is ongoing. However most evaluations are quite fast (i.e. only on a short interval and one epoch), this is not so useful.

Also when now we batch the evaluation requests, it is more tricky to keep these fields correct. To balance the usefulness and efforts, I decided to remove them.

To verify the evaluation actually contains expected data, in `EvaluationModelResponse` there is a field `dataset_sizes`. To verify the progress is going correctly, we still have the log https://github.com/eth-easl/modyn/blob/60d3baa7523d21197dec85262fb8c31662d656c2/modyn/evaluator/internal/pytorch_evaluator.py#L168.

Reply:
Hmm, stdout logs are virtually useless when running big experiments with a lot of evaluations and you don't have an initial idea that sth. would be wrong.
Currently, we are using this information in the analytics tool to e.g. check that certain evaluation strategy configurations did what they are supposed to do.

Reply:
I see. Can you point me where in the code how any field is used? I am happy to add them back and implement correctly, just to understand their usage.

Reply:
anyway, I will add back `batches_seen` and `samples_seen`, together with a new field `current_interval_idx`

Reply:
I just checked I think we only use `dataset_size` ([here](https://github.com/eth-easl/modyn/blob/60d3baa7523d21197dec85262fb8c31662d656c2/modyn/supervisor/internal/pipeline_executor/models.py#L318C1-L319C1)). Sorry for the confusion, I don't know what the semantics of these two fields is then. If you don't find them necessary I'm happy to drop them

Reply:
okay great! then I will just ask Maxi. Thanks for confirming this.